src/js/apps/patients/worklist/filters_app.js

```diff
@@ -86,8 +88,8 @@ export default App.extend({
     });
 
     this.listenTo(dateFilterComponent.getState(), {
-      'change'(state) {
-        this.setState(state);
+      'change'({ attributes }) {
```

Comment:
I think this was technically broken, but either way needed the clone to trigger a change event after the 1st change

Reply:
Oooh, you're referring to the filters app state change event right? So the change event listener in worklist_app wasn't getting fired?

Reply:
well the original line was setting the state to the state model and not it's attributes, I changed it to attributes but it would only get the first change event fired because every subsequent update was just updating the attributes object with the same object